frame/aura/src/lib.rs

```diff
@@ -60,6 +60,20 @@ pub use pallet::*;
 
 const LOG_TARGET: &str = "runtime::aura";
 
+/// A slot duration provider which infers the slot duration from the
+/// [`pallet_timestamp::Config::MinimumPeriod`] by multiplying it by two, to ensure
+/// that authors have the majority of their slot to author within.
+///
+/// This was the default behavior of the Aura pallet and may be used for
+/// backwards compatibility.
+pub struct MinimumPeriodTimesTwo<T>(sp_std::marker::PhantomData<T>);
```

Comment:
This doesn't seem to be used anywhere, maybe you meant to use it in `fn slot_duration`?

Reply:
No, it's strictly necessary with the explicit-slot-duration feature. I don't see a reason this should be feature flagged as it's strictly additive.